### DISCUSSION TITLE
Please add dark theme

### DISCUSSION THREAD

**Discussion by jardicc** [npm/feedback#231] [Website] [answered]
I do have a dark theme for StackOverflow, VSCode, Slack, GitHub, Chrome DevTools, Windows even Total Commander, and many other places but NPM is still shining bright :-) Please consider adding a dark theme.

Perhaps something like this: 
![image](https://user-images.githubusercontent.com/13526959/108224271-040e7e80-713b-11eb-9196-b182006b3932.png)
(source: https://userstyles.org/styles/169643/npm-dark-theme-complete)

**Comment by MylesBorins** [comment, marked as answer]
Thanks for the suggestion @jardicc 

At this time the npm team will no be able to prioritize dark mode. While it is definitely a great feature we have a [pretty full roadmap](https://github.com/npm/roadmap/projects/1) and are focusing on smaller incremental improvements.

**Comment by ilfa** [comment]
It looks like we can't [Specify the theme an image is shown to](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to) for the readme images that will be published in npm while it hasn't a dark theme.

**Comment by movahhedi** [comment]
Hi, How you're doing well.
As a developer, I work with NPM every day. Every single tool of mine is in dark mode, from Windows, to chatting apps, to dev tools. I even switched from Windows 10 to 11 JUST for the dark mode Notepad. Everything, but NPM. My eyes really hurt when I read READMEs. I even go to the repo to read it in GitHub's dark mode.
So please, add it. If needed, I can help with it too, just ping me.
But, really; please.
